File: Examples/VisualBasic/AsposePdfFacades/Text/ReplaceText.vb
Imports System.IO
Imports System
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades
Namespace AsposePDFFacades.Text
    Public Class ReplaceText
        Public Shared Sub Run()
            ' ExStart:ReplaceText
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_Text()
            ' Open input PDF
            Dim pdfContentEditor As New PdfContentEditor()
            pdfContentEditor.BindPdf(dataDir & Convert.ToString("ReplaceText.pdf"))
            ' Replace text on all pages
            pdfContentEditor.ReplaceText("Hello", "World")
            ' Save output PDF
            pdfContentEditor.Save(dataDir & Convert.ToString("ReplaceText_out_.pdf"))
            ' ExEnd:ReplaceText
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePdfFacades/Text/ReplaceText.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;

namespace Aspose.Pdf.Examples.CSharp.AsposePDFFacades.Text
{
    public class ReplaceText
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_Text();
            //open input PDF
            PdfContentEditor pdfContentEditor = new PdfContentEditor();
            pdfContentEditor.BindPdf(dataDir+ "input1.pdf");
            //replace text on all pages
            pdfContentEditor.ReplaceText("Hello", "World");
            //save output PDF
            pdfContentEditor.Save(dataDir+ "ReplaceTextOnAllPages.pdf");
            
        }
    }
}